Person.rb

```diff
@@ -7,14 +7,15 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Great job using getters for your rentals. however, please kindly take note that the project requirement is to use Getters and Setters for @rentals. please kindly make use of attr_accessor for your rentals also.